cmake/dts.cmake

```diff
@@ -10,6 +10,9 @@
 # See ~/zephyr/doc/dts
 set(GENERATED_DTS_BOARD_H    ${PROJECT_BINARY_DIR}/include/generated/generated_dts_board.h)
 set(GENERATED_DTS_BOARD_CONF ${PROJECT_BINARY_DIR}/include/generated/generated_dts_board.conf)
+
+include(${BOARD_DIR}/dts.cmake OPTIONAL)
+
```

Comment:
This should be in a dedicated PR.

Reply:
This PR has 2 separate commits. One of them is only for this change. Still need separate PR for this one?

Reply:
No, I do not think we need another PR for this as long as it is split into different commits. Thanks.